tests/tracking/fluent/test_fluent.py

```diff
@@ -1194,3 +1195,11 @@ def test_set_experiment_tags():
     assert len(finished_experiment.tags) == len(exact_expected_tags)
     for tag_key, tag_value in finished_experiment.tags.items():
         assert str(exact_expected_tags[tag_key]) == tag_value
+
+
+def test_get_metric_history_on_non_existent_metric_key():
```

Comment:
This test should be moved to `mlflow.tests.tracking.test_rest_tracking` to validate the behavior of the rest store. Should be a quick lift-and-ship :)

Reply:
Well, specifically, not the function, but the validation logic in this test implementation. Add a check for a non-existent metric key to one of the suites :)